tests/migration/ember-addon/steps/use-relative-paths/base-case.test.js

```diff
@@ -0,0 +1,163 @@
+import { useRelativePaths } from '../../../../../src/migration/ember-addon/steps/use-relative-paths.js';
+import { assertFixture, loadFixture, test } from '../../../../test-helpers.js';
+
+const fileMapping = new Map([
+  [
+    'addon/components/container-query.ts',
+    {
+      input: [
+        `import { action } from '@ember/object';`,
+        `import Component from '@glimmer/component';`,
+        `import { tracked } from '@glimmer/tracking';`,
+        ``,
+        `import type {`,
+        `  Dimensions,`,
+        `  Features,`,
+        `  IndexSignatureParameter,`,
+        `  QueryResults,`,
+        `} from 'ember-container-query/modifiers/container-query';`,
+      ].join('\n'),
+
+      output: [
+        `import { action } from '@ember/object';`,
+        `import Component from '@glimmer/component';`,
+        `import { tracked } from '@glimmer/tracking';`,
+        ``,
+        `import type {`,
+        `  Dimensions,`,
+        `  Features,`,
+        `  IndexSignatureParameter,`,
+        `  QueryResults,`,
+        `} from '../modifiers/container-query';`,
+      ].join('\n'),
+    },
+  ],
+
+  [
+    'tests/dummy/app/components/widgets/widget-3/tour-schedule/responsive-image.ts',
+    {
+      input: [
+        `import { action } from '@ember/object';`,
+        `import Component from '@glimmer/component';`,
+        `import { tracked } from '@glimmer/tracking';`,
+        `import type { Image } from 'dummy/data/concert';`,
+        `import { findBestFittingImage } from 'dummy/utils/components/widgets/widget-3';`,
+        `import type { Dimensions } from 'ember-container-query/modifiers/container-query';`,
+      ].join('\n'),
+
+      output: [
+        `import { action } from '@ember/object';`,
+        `import Component from '@glimmer/component';`,
+        `import { tracked } from '@glimmer/tracking';`,
+        `import type { Image } from '../../../../data/concert';`,
+        `import { findBestFittingImage } from '../../../../utils/components/widgets/widget-3';`,
```

Comment:
Here, we're asserting that `useRelativePaths` ...

- Finds and replaces multiple "magic" paths